content/en/1.13.x/concepts/updating-bottlerocket.markdown

```diff
@@ -0,0 +1,299 @@
+# Updating Bottlerocket
+
+This document covers the different ways to update Bottlerocket (for [ECS](#ecs), [Kubernetes](#kubernetes), and [SSM](#ssm)), as well as how to [lock your nodes to a specific release](#locking-to-a-specific-release).
+
+## Introduction
+
+Bottlerocket is designed to be updated in an [image-based fashion](https://github.com/bottlerocket-os/bottlerocket#updates).
+This means that updates are applied by replacing the entire image on disk, rather than applying a series of individual package updates.
```

Comment:
This should mention that we flip to a different partition. https://github.com/bottlerocket-os/bottlerocket/blob/e35379a49e3ab9533380922cc1124e1ffee7ba1d/README.md?plain=1#L21

Reply:
Agreed. How does the following sound?:

```
This means that updates are applied by downloading an image of the entire operating system to a different partition on disk, then switching to using that partition when the system is rebooted. This is done instead of a series of individual package updates on the current operating system partition.
```